src/components/atoms/Dropdown/Dropdown.scss

```diff
@@ -0,0 +1,30 @@
+@import "scss/constants";
+
+.Dropdown {
+  .dropdown-toggle.btn {
+    background: none;
+    padding: 0;
+    margin: 0;
+    border: none;
+    line-height: unset;
+    vertical-align: baseline;
+
+    &:focus {
+      box-shadow: none;
+    }
+
+    &:hover {
+      background-color: initial;
+      transform: none;
+    }
+  }
+
+  .dropdown-item {
```

Comment:
`&__item` ?

Reply:
fixed in f52680a3d71cea4dbb393174dc4e36cd9455407d